components/Buttons/MarkPlayedButton.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-btn v-if="canMarkWatched" icon dark @click.stop.prevent="togglePlayed">
+    <v-icon :color="isPlayed ? 'primary' : 'white'"> mdi-check </v-icon>
```

Comment:
```suggestion
    <v-icon :color="isPlayed ? 'primary' : 'white'">mdi-check</v-icon>
```

This isn't fixed automatically because the brace rule?

Reply:
we seem to have a mix of both `> content<` and `>content<` throught the codebase